Dockerfile.viewer

```diff
@@ -0,0 +1,57 @@
+FROM raspbian/stretch
+MAINTAINER Anton Molodykh <amolodykh@screenly.io>
+
+RUN apt-get update && \
+    apt-get -y install \
+        build-essential \
+        curl \
+        git-core \
+        gstreamer0.10-plugins-good \
+        libdbus-glib-1-dev \
+        libffi-dev \
+        libqt5webkit5 \
```

Comment:
Do we need this given that we have compiled the binary in the other repo?

Reply:
yes, the binary file has static links to the files from these libraries